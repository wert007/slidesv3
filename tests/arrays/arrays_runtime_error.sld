
func main() {
    let a = [1, 2, 3];
    print(a[99]);
    a[99] = -55;
}