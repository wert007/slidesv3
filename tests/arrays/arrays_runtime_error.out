Runtime error happened in std.sld:163:13: Index 99 is out of bounds of array with length 3.
  ...in ..\tests\arrays\arrays_runtime_error.sld:1:1
  ...in ..\tests\arrays\arrays_runtime_error.sld:4:11
  ...in std.sld:168:9
  ...in std.sld:163:13
Unusual termination.
= -1
